static/docs/user-guide/using-docker.md

```diff
@@ -0,0 +1,108 @@
+# Using Docker
+
+Currently we don't provide any official Docker image, however you can easily
+create it yourself.
+
+## Writing the Dockerfile
+
+We support several ways to install DVC, the two recommended ones are through
+`pip` and `conda`.
+
+You can base your image on the official Python one:
+```dockerfile
+FROM python
+
+# Remember to include the cloud providers that you plan to use,
+# for example: "dvc[s3]"
+RUN pip install dvc
+```
+
+Or use a conda based image:
+```dockerfile
+FROM continuumio/miniconda3
+
+RUN conda install dvc -y --quiet
+```
+
+For a recommended reading about writing Dockerfiles for Python applications,
+you can take a look at https://pythonspeed.com/docker/
+
+## Using Docker for your development environment
+
+Since `dvc` is a command line application, we have several recommendiations
+to enhance the experience:
+  * Make sure your _locale_ is set to UTF-8
+  * Install the `Zsh` shell and its respective autocompletion script.
+  * Install `less` so you can pipe the output of `dvc pipeline show`.
+  * Enable movements with `Ctrl + Arrow` by editing the _inputrc_ file.
+  * Use `pip install dvc` at the end of the file for better caching.
+  * Set your `WORKDIR` to the path where you are going to mount your project.
+
+If you are using Linux, remember to map your user to your container. This way,
+you make sure your files are still writtable. For more information:
+https://docs.docker.com/engine/security/userns-remap/
+
+### Dockerfile
+
+The following is a suggestion for a Dockerfile to use during development:
```

Comment:
we need to update SyntaxHighlighter in the `Markdown.js` to enable Docker. I hope it's available.

Reply:
looks like it is)

https://github.com/conorhastings/react-syntax-highlighter/blob/d761799ef25304864504d427dad92f0c2c1486cb/src/languages/hljs/dockerfile.js